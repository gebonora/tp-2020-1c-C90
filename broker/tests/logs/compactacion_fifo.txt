Partici贸n 1: | 0 - 19 | [X] | Size: 20 b | LRU: 13 | COLA: CATCH | ID: 4 |
Partici贸n 2: | 20 - 35 | [X] | Size: 16 b | LRU: 16 | COLA: CATCH | ID: 5 |
Partici贸n 3: | 36 - 55 | [X] | Size: 20 b | LRU: 24 | COLA: CATCH | ID: 7 |
Partici贸n 4: | 56 - 63 | [L] | Size: 8 b | LRU: 23 |
