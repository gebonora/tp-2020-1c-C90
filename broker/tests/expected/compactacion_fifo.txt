Partici贸n 1: | 0 - 19 | [X] | Size: 20 b | LRU: 4 | COLA: CATCH | ID: 4 |
Partici贸n 2: | 20 - 35 | [X] | Size: 16 b | LRU: 5 | COLA: CATCH | ID: 5 |
Partici贸n 3: | 36 - 55 | [X] | Size: 20 b | LRU: 7 | COLA: CATCH | ID: 6 |
Partici贸n 4: | 56 - 63 | [L] | Size: 8 b |
